hw/xbox/nv2a.c

```diff
@@ -1309,11 +1311,14 @@ typedef struct VertexAttribute {
     unsigned int converted_size;
     unsigned int converted_count;
 
+    float *inline_buffer_buffer;
```

Comment:
gl_inline_buffer_buffer is named that because it's a opengl buffer for inline_buffer ;)